frontend/src/app/master-page.module.ts

```diff
@@ -153,6 +154,7 @@ export class MasterPageRoutingModule { }
   ],
   declarations: [
     MasterPageComponent,
+    AngorProjectsComponent,
```

Comment:
again also here this angor already works on mempool so why do we need to define it here? it should be already defined no?